src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
```

Comment:
I don't think this assembly should have any dependencies on MSBuild, unless it is 100% necessary.